registration/views.py

```diff
@@ -516,7 +516,18 @@ def done(self, form_list, **kwargs):
         if self.request.session[SESSION_REGISTRATION_UPDATE]:
             return django.shortcuts.redirect('update_done')
         else:
-            return django.shortcuts.redirect('email_next_of_kin')
+            return self.get_next_of_kin_page()
+
+    def get_next_of_kin_page(self):
+        try:
+            count = cache.incr('done_registration_count')
+        except ValueError:
+            count = 0
+            cache.set('done_registration_count', 0)
+        if count % 2:
+            return django.shortcuts.redirect('email_next_of_kin_1')
+        else:
+            return django.shortcuts.redirect('email_next_of_kin_2')
```

Comment:
Why not do:

```
NOK_PAGE_NAME_CHOICES = ['email_next_of_kin_a', 'email_next_of_kin_b', ]  # this should be defined as a constant at the top of the page
return django.shortcuts.redirect(random.choice(NOK_PAGE_NAME_CHOICES))
```

Much fewer lines of code and doesn't require redis. We don't care if the counts are always 50% each, just that they are statistically randomized.